### PR TITLE
#163348045 Fixes failing build 

### PR DESCRIPTION
**Pull request by Felix45**
**What does this pull request do**
- Fixes Travis CI build fail

**Description of the task to be completed**
- Fix application configurations and bugs to enable Travis CI build,  pass

**How should this be manually tested**
- Clone or fork the repository 
`$ git clone https://github.com/Felix45/Questioner.git`

- Open a terminal and change the working directory to Questioner
`$ cd Questioner/app/tests/v1`

- Install a virtual environment
`$ virtualenv env`

- Once the environment is up, activate it using
`$ source env/bin/activate`

- Install all the requirements for the application using
`$ pip install -r requirements.txt`

- Checkout to branch bg-fix-failing-build-163348045
`$ git checkout bg-fix-failing-build-163348045`

**Relevant pivotal tracker stories**
[Fix Travis CI build error](https://www.pivotaltracker.com/n/projects/2235209/stories/163127427)

**Screen shots**
current-tests

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,4 +2,6 @@ env/
 *~
 *.pyc
 __pycache__/
-.vscode/
\ No newline at end of file
+.vscode/
+*.coverage
+*.env
```
Comment (RuthNjeri) [inline .gitignore]: add an extra line after line 7 to create a `newline at the end of the file`

File: app/api/v1/models/questions_model.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    def __init__(self):
+        self.questions = []
+        self.helpers = UsersHelper()
+
+    def add_a_question(self, user_request):
+        """ Add a question record """
+        question = {}
+        keys_expected = ["user", "meetup", "title", "body"]
+
+        ret = self.helpers.is_valid_user_request(keys_expected, user_request)
```
Comment (RuthNjeri) [inline app/api/v1/models/questions_model.py]: Please use descriptive variable names to describe what it is doing.

File: app/api/v1/models/questions_model.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        question['title'] = user_request.get_json()['title']
+        question['body'] = user_request.get_json()['body']
+
+        self.questions.append(question)
+        return jsonify({"msg": "question was added", "data":self.questions, "status": 201}), 201
+
+    def vote(self, vote, question_id, type):
+        """ Allows a user to upvote or downvote a question """
+
+        question = [q for q in self.questions if q["id"] == question_id]
```
Comment (RuthNjeri) [inline app/api/v1/models/questions_model.py]: use descriptive variable names as opposed to `q`

File: app/api/v1/models/questions_model.py [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    
+    def get_a_question(self, search_id):
+        """ Returns a specific question record """
+
+        if len(self.questions) == 0:
+            return jsonify({"msg": "no questions were found", "data": self.questions, "status": 200}), 200      
+        for question in self.questions:
+            if question['id'] == search_id:
+                return jsonify({"msg": "question was found", "data": question, "status": 200})     
+        return jsonify({"msg": "question was not found", "data": [], "status": 404}), 404
```
Comment (RuthNjeri) [inline app/api/v1/models/questions_model.py]: add a new line at the end of this file

File: app/api/v2/views/meetups_view.py [hunk 5]
```diff
@@ -1,13 +1,16 @@
...
+from app.api.v2.models.meetups_model import UsersModel
 
 meetupV2 = Blueprint('meetup_v2', __name__, url_prefix='/api/v2')
 
 meetups_model = MeetUpsModel()
+users_model = UsersModel()
 
 
 @meetupV2.route('/meetups', methods=['POST'])
+#@users_model.token_required
```
Comment (RuthNjeri) [inline app/api/v2/views/meetups_view.py]: Do not push commented code. If you are not using it then remove it.

**Comment by RuthNjeri** [comment]
I noticed your Git commit messages are in the past tense, this is not the best convention. Please refer to this resource on the best way to write a commit message.
screenshot 2019-01-22 at 21 29 44
https://chris.beams.io/posts/git-commit/

**Comment by Felix45** [comment]
> I noticed your Git commit messages are in the past tense, this is not the best convention. Please refer to this resource on the best way to write a commit message.
> screenshot 2019-01-22 at 21 29 44
> https://chris.beams.io/posts/git-commit/

> I noticed your Git commit messages are in the past tense, this is not the best convention. Please refer to this resource on the best way to write a commit message.
> screenshot 2019-01-22 at 21 29 44
> https://chris.beams.io/posts/git-commit/

I've gone through this article and I will improve on how I write my commit messages

**Comment by Felix45** [comment]
> > I noticed your Git commit messages are in the past tense, this is not the best convention. Please refer to this resource on the best way to write a commit message.
> > screenshot 2019-01-22 at 21 29 44
> > https://chris.beams.io/posts/git-commit/
> 
> > I noticed your Git commit messages are in the past tense, this is not the best convention. Please refer to this resource on the best way to write a commit message.
> > screenshot 2019-01-22 at 21 29 44
> > https://chris.beams.io/posts/git-commit/
> 
> I've gone through this article and I will improve on how I write my commit messages. The article was very helpul and informative.

File: app/api/v2/views/meetups_view.py (hunk 5 above)
Comment (Felix45) [reply app/api/v2/views/meetups_view.py]: I've Implemented the changes recommended on branch name bg-fix-failing-build-163348045

File: app/api/v1/models/questions_model.py (hunk 4 above)
Comment (Felix45) [reply app/api/v1/models/questions_model.py]: I've Implemented the changes recommended on branch name bg-fix-failing-build-163348045

File: app/api/v1/models/questions_model.py (hunk 3 above)
Comment (Felix45) [reply app/api/v1/models/questions_model.py]: I've Implemented the changes recommended on branch name bg-fix-failing-build-163348045

**Pull request merged**